.ci/integration_test.sh

```diff
@@ -236,8 +237,43 @@ integration_gm()
     LOG_INFO "integration_gm testing pass."
 }
 
+integration_amop() {
+    LOG_INFO "integration_amop testing..."
+    bash nodes/127.0.0.1/start_all.sh
+
+    execute_cmd "go build -o subscriber examples/amop/sub/subscriber.go"
+    execute_cmd "go build -o unicast_publisher examples/amop/unicast_pub/publisher.go"
+    nohup ./unicast_publisher 127.0.0.1:20200 hello > output.file 2>&1 &
+    nohup ./subscriber 127.0.0.1:20201 hello > subscriber0.out 2>&1 &
+    sleep 10s
+    grep "hello, FISCO BCOS" ./subscriber0.out >> /dev/null
+    if [[ $? -ne 0 ]];then LOG_ERROR "amop unique broadcast failed." && exit 1;fi
+    pid=$(ps -ef | grep -v grep | grep unicast_publisher | awk '{print $2}')
+    if [[ ! -z ${pid} ]];then kill -9 ${pid};fi
+    pid=`ps -ef | grep -v grep | grep subscriber | awk '{print $2}'`
```

Comment:
Codacy found an issue: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6008073)